### ISSUE TITLE
Legacy test failures

### ISSUE THREAD

**Issue by mxz297** [dyninst/dyninst#550]
The Dyninst test suite has a couple of test failures that have been around for a long time, ranging from a few months to several years. 

If the test itself is wrong, the test should be disabled. Otherwise, I will try to fix Dyninst.

Here is the list for x86-64:

```
test_pt_ls                        none 64  create   NA      dynamic nonPIC  FAILED (Running Test)
test_pt_ls                        none 64  rewriter NA      dynamic nonPIC  FAILED (Running Test)
test1_22                   g++    none 64  rewriter NA      dynamic PIC     FAILED (Group Teardown)
pc_tls                     g++    none 64  create   SPST    dynamic nonPIC  FAILED (Running Test)
```

**Comment by sashanicolas** [comment]
for aarch64:
```
TEST                       COMP   OPT  ABI MODE     THREAD  LINK    PIC     RESULT
test_pt_ls                        none 64  create   NA      dynamic nonPIC  FAILED (Running Test)
test_pt_ls                        none 64  attach   NA      dynamic nonPIC  SKIPPED
test_pt_ls                        none 64  rewriter NA      dynamic nonPIC  FAILED (Running Test)
```

**Comment by mxz297** [comment]
@sashanicolas I found that test_pt_ls failed because it cannot find parseThat. I updated the test suite to properly search for parseThat and the test_pt_ls is now passing on my machine. Can you pull the test suite and run it on leela?

**Comment by hainest** [comment]
Fixed by #549
